Makefile

```diff
@@ -101,3 +103,24 @@ docs:
 
 build_spl: clean
 	python -m splunk_eventgen build --destination ./
+
+lint:
+ifeq ($(NEWLY_ADDED_PY_FILES), )
+	@echo 'No newly added python files. Skip...'
+else
+	@flake8 $(NEWLY_ADDED_PY_FILES) || true
+endif
+	@git diff -U0 -- '*.py' | flake8 --diff || true
+
```

Comment:
`make format` will only format newly added python files and  sort imports for both newly added and changed python files.